app/models/manageiq/providers/google/inventory/parser.rb

```diff
@@ -9,6 +9,10 @@ class ManageIQ::Providers::Google::Inventory::Parser < ManageIQ::Providers::Inve
     "UNHEALTHY" => "OutOfService"
   }.freeze
 
+  def options
```

Comment:
NOTE Options are available in the base `Inventory::Collector` class [[link]](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/inventory/collector.rb#L17-L20) and I think they should be in the base persister class as well, will drop this pending merge of a pending core PR :wink:

Reply:
https://github.com/ManageIQ/manageiq/pull/21460

Reply:
Merged ManageIQ/manageiq#21460